airbyte-cdk/java/airbyte-cdk/db-sources/src/testFixtures/java/io/airbyte/cdk/integrations/debezium/CdcSourceTest.java

```diff
@@ -48,153 +47,159 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public abstract class CdcSourceTest {
+public abstract class CdcSourceTest<S extends Source, T extends TestDatabase<?, T, ?>> {
```

Comment:
This class is rewritten so that the only shared state between test methods is `protected T testdb`. The test logic itself remains unchanged.